src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -8,10 +8,48 @@
 					xmlns:ut="using:Uno.Themes"
 					mc:Ignorable="d macos">
 
+	<!--
+		Expected TextBox behavior:
+		We are going a different route than the Official Material M2/M3 Google samples & the Material doc regarding the fact that the Placeholder will always be visible.
+		(cf. specs discussions here https://github.com/unoplatform/Uno.Themes/issues/974)
+		
+		- Without Header/Placeholder, we will have a simple TextBox (matching Windows behavior)
+		
+		- With a Placeholder and no Header, we will have a simple TextBox (matching Windows behavior).
+		
+		- If there is a Header:
+		- Without a Placeholder, the Header moves up when focused and it will be empty underneath until some text is entered
+		- With a Placeholder, the Header does not move, it is already at the top and shows also the Placeholder underneath
+	-->
+
 	<!-- Converters -->
-	<ut:FromEmptyStringToValueConverter x:Key="EmptyToCompositeTransformTranslateY"
-										NotNullOrEmptyValue="8"
-										NullOrEmptyValue="0" />
+	<um:FromTextBoxEmptyStringToValueConverter x:Key="PlaceholderContentCompositeTransformTranslateY"
+																						 HeaderAndPlaceholderValue="8"
```

Comment:
any chance that you are using tab with a size of 2?

Reply:
Not me, but the solution, apparently: https://github.com/unoplatform/Uno.Themes/blob/6b8beaaacb68d120db4e9bf9a3b5c48aaef4c11e/.editorconfig#L44

I tried to follow the pattern:

![image](https://github.com/unoplatform/Uno.Themes/assets/114886335/eed4fe2d-4748-4143-8daa-e202b68e822f)

Reply:
not anymore soon: #1417

Reply:
That's fixed now